src/js/utils/browser.js

```diff
@@ -177,5 +177,10 @@ define([
         return majorVersion;
     };
 
+    browser.nativeRenderingSupported = function (providerName) {
+        return providerName.indexOf('flash') === -1 &&
+        (browser.isChrome() || browser.isIOS() || browser.isSafari() || browser.isEdge());
+    };
+
```

Comment:
This should not be a util. Ideally this would be set on the config level, and the default setting would be based on these rules.